2024-01-10 16:55:03,967 INFO    StreamThr :2597697 [internal.py:wandb_internal():86] W&B internal server running at pid: 2597697, started at: 2024-01-10 16:55:03.966761
2024-01-10 16:55:03,977 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: status
2024-01-10 16:55:03,991 INFO    WriterThread:2597697 [datastore.py:open_for_write():85] open: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/run-4ffetev3.wandb
2024-01-10 16:55:03,992 DEBUG   SenderThread:2597697 [sender.py:send():380] send: header
2024-01-10 16:55:04,179 DEBUG   SenderThread:2597697 [sender.py:send():380] send: run
2024-01-10 16:55:05,044 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: check_version
2024-01-10 16:55:05,050 INFO    SenderThread:2597697 [dir_watcher.py:__init__():211] watching files in: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files
2024-01-10 16:55:05,050 INFO    SenderThread:2597697 [sender.py:_start_run_threads():1122] run started: 4ffetev3 with start time 1704876903.975909
2024-01-10 16:55:05,050 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: check_version
2024-01-10 16:55:05,399 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: run_start
2024-01-10 16:55:05,407 DEBUG   HandlerThread:2597697 [system_info.py:__init__():32] System info init
2024-01-10 16:55:05,407 DEBUG   HandlerThread:2597697 [system_info.py:__init__():47] System info init done
2024-01-10 16:55:05,407 INFO    HandlerThread:2597697 [system_monitor.py:start():194] Starting system monitor
2024-01-10 16:55:05,407 INFO    SystemMonitor:2597697 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-01-10 16:55:05,408 INFO    HandlerThread:2597697 [system_monitor.py:probe():214] Collecting system info
2024-01-10 16:55:05,408 INFO    SystemMonitor:2597697 [interfaces.py:start():190] Started cpu monitoring
2024-01-10 16:55:05,408 INFO    SystemMonitor:2597697 [interfaces.py:start():190] Started disk monitoring
2024-01-10 16:55:05,408 INFO    SystemMonitor:2597697 [interfaces.py:start():190] Started gpu monitoring
2024-01-10 16:55:05,409 INFO    SystemMonitor:2597697 [interfaces.py:start():190] Started memory monitoring
2024-01-10 16:55:05,409 INFO    SystemMonitor:2597697 [interfaces.py:start():190] Started network monitoring
2024-01-10 16:55:05,429 DEBUG   HandlerThread:2597697 [system_info.py:probe():196] Probing system
2024-01-10 16:55:05,433 DEBUG   HandlerThread:2597697 [system_info.py:_probe_git():181] Probing git
2024-01-10 16:55:05,438 DEBUG   HandlerThread:2597697 [system_info.py:_probe_git():189] Probing git done
2024-01-10 16:55:05,438 DEBUG   HandlerThread:2597697 [system_info.py:probe():244] Probing system done
2024-01-10 16:55:05,438 DEBUG   HandlerThread:2597697 [system_monitor.py:probe():223] {'os': 'Linux-5.15.0-46-generic-x86_64-with-glibc2.17', 'python': '3.8.13', 'heartbeatAt': '2024-01-10T08:55:05.429764', 'startedAt': '2024-01-10T08:55:03.954695', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': 'train.py', 'codePathLocal': 'train.py', 'codePath': 'train.py', 'git': {'remote': None, 'commit': '18c7fe0993c97a9f7a0f2252b6f7c1dee73e5c87'}, 'email': 'none', 'root': '/home/shangjian/code/Research/Multimodal & LLM/SLM', 'host': 'scau540-Song', 'username': 'shangjian', 'executable': '/home/shangjian/miniconda3/envs/pro/bin/python', 'cpu_count': 6, 'cpu_count_logical': 12, 'cpu_freq': {'current': 3.8426666666666662, 'min': 2200.0, 'max': 3500.0}, 'cpu_freq_per_core': [{'current': 3.5, 'min': 2200.0, 'max': 3500.0}, {'current': 4.449, 'min': 2200.0, 'max': 3500.0}, {'current': 3.5, 'min': 2200.0, 'max': 3500.0}, {'current': 4.449, 'min': 2200.0, 'max': 3500.0}, {'current': 3.5, 'min': 2200.0, 'max': 3500.0}, {'current': 3.5, 'min': 2200.0, 'max': 3500.0}, {'current': 3.5, 'min': 2200.0, 'max': 3500.0}, {'current': 3.5, 'min': 2200.0, 'max': 3500.0}, {'current': 3.816, 'min': 2200.0, 'max': 3500.0}, {'current': 4.449, 'min': 2200.0, 'max': 3500.0}, {'current': 3.5, 'min': 2200.0, 'max': 3500.0}, {'current': 4.449, 'min': 2200.0, 'max': 3500.0}], 'disk': {'/': {'total': 5498.824867248535, 'used': 2542.0863189697266}}, 'gpu': 'NVIDIA GeForce RTX 3090', 'gpu_count': 2, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3090', 'memory_total': 25769803776}, {'name': 'NVIDIA GeForce RTX 3090', 'memory_total': 25769803776}], 'memory': {'total': 62.7276725769043}}
2024-01-10 16:55:05,438 INFO    HandlerThread:2597697 [system_monitor.py:probe():224] Finished collecting system info
2024-01-10 16:55:05,438 INFO    HandlerThread:2597697 [system_monitor.py:probe():227] Publishing system info
2024-01-10 16:55:05,438 DEBUG   HandlerThread:2597697 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-01-10 16:55:05,438 DEBUG   HandlerThread:2597697 [system_info.py:_save_pip():68] Saving pip packages done
2024-01-10 16:55:05,439 DEBUG   HandlerThread:2597697 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-01-10 16:55:06,051 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_created():271] file/dir created: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/conda-environment.yaml
2024-01-10 16:55:06,052 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_created():271] file/dir created: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/requirements.txt
2024-01-10 16:55:06,207 DEBUG   HandlerThread:2597697 [system_info.py:_save_conda():87] Saving conda packages done
2024-01-10 16:55:06,207 INFO    HandlerThread:2597697 [system_monitor.py:probe():229] Finished publishing system info
2024-01-10 16:55:06,212 DEBUG   SenderThread:2597697 [sender.py:send():380] send: files
2024-01-10 16:55:06,213 INFO    SenderThread:2597697 [sender.py:_save_file():1378] saving file wandb-metadata.json with policy now
2024-01-10 16:55:06,217 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: stop_status
2024-01-10 16:55:06,217 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: stop_status
2024-01-10 16:55:06,219 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-10 16:55:06,744 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/520/huggingface/4ffetev3/wandb-metadata.json?Expires=1704963306&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=I6KM711nl5VWVYhSgQZFEaH%2FWHWikDnRWjFVqNHXw%2BpUY5MNXLI4pMGpQEaYHSaVRX%2FDglipac46Z6uvIj6%2B6zO%2FKI6nbibE3uP1GcxYhpYQtMaop%2BQ5j4jD85%2B%2Brd97KCS9brRTziub7K602SkszBDBCphMYktmBkby3FJeceL0RvwEHQ8YBca%2FrPaNnX1rjTLIt68f2JHnRUw3CRIWFEg2QXrlS1h0%2BiPev3cJEVkppZ349bvZA5U1Jw%2FplFTVnTz3JELy%2BrNgbNmyqMtcfshccW1KzSoqu%2BVAmzKYjKni8uRhgAoPY9GR4jkqd6iFh%2FauSv7jzx%2FdhJlqCXTWgw%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-01-10 16:55:06,744 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2573'}
2024-01-10 16:55:06,744 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2560] upload_file response body: 
2024-01-10 16:55:06,891 DEBUG   SenderThread:2597697 [sender.py:send():380] send: telemetry
2024-01-10 16:55:06,891 DEBUG   SenderThread:2597697 [sender.py:send():380] send: config
2024-01-10 16:55:06,892 DEBUG   SenderThread:2597697 [sender.py:send():380] send: metric
2024-01-10 16:55:06,892 DEBUG   SenderThread:2597697 [sender.py:send():380] send: telemetry
2024-01-10 16:55:06,892 DEBUG   SenderThread:2597697 [sender.py:send():380] send: metric
2024-01-10 16:55:06,892 WARNING SenderThread:2597697 [sender.py:send_metric():1329] Seen metric with glob (shouldn't happen)
2024-01-10 16:55:06,892 DEBUG   SenderThread:2597697 [sender.py:send():380] send: telemetry
2024-01-10 16:55:07,051 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/conda-environment.yaml
2024-01-10 16:55:07,051 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_created():271] file/dir created: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/wandb-metadata.json
2024-01-10 16:55:07,051 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_created():271] file/dir created: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:07,816 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/520/huggingface/4ffetev3/wandb-metadata.json?Expires=1704963306&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=I6KM711nl5VWVYhSgQZFEaH%2FWHWikDnRWjFVqNHXw%2BpUY5MNXLI4pMGpQEaYHSaVRX%2FDglipac46Z6uvIj6%2B6zO%2FKI6nbibE3uP1GcxYhpYQtMaop%2BQ5j4jD85%2B%2Brd97KCS9brRTziub7K602SkszBDBCphMYktmBkby3FJeceL0RvwEHQ8YBca%2FrPaNnX1rjTLIt68f2JHnRUw3CRIWFEg2QXrlS1h0%2BiPev3cJEVkppZ349bvZA5U1Jw%2FplFTVnTz3JELy%2BrNgbNmyqMtcfshccW1KzSoqu%2BVAmzKYjKni8uRhgAoPY9GR4jkqd6iFh%2FauSv7jzx%2FdhJlqCXTWgw%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-01-10 16:55:07,816 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2573'}
2024-01-10 16:55:07,816 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2560] upload_file response body: 
2024-01-10 16:55:09,051 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:09,582 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: status_report
2024-01-10 16:55:09,893 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/520/huggingface/4ffetev3/wandb-metadata.json?Expires=1704963306&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=I6KM711nl5VWVYhSgQZFEaH%2FWHWikDnRWjFVqNHXw%2BpUY5MNXLI4pMGpQEaYHSaVRX%2FDglipac46Z6uvIj6%2B6zO%2FKI6nbibE3uP1GcxYhpYQtMaop%2BQ5j4jD85%2B%2Brd97KCS9brRTziub7K602SkszBDBCphMYktmBkby3FJeceL0RvwEHQ8YBca%2FrPaNnX1rjTLIt68f2JHnRUw3CRIWFEg2QXrlS1h0%2BiPev3cJEVkppZ349bvZA5U1Jw%2FplFTVnTz3JELy%2BrNgbNmyqMtcfshccW1KzSoqu%2BVAmzKYjKni8uRhgAoPY9GR4jkqd6iFh%2FauSv7jzx%2FdhJlqCXTWgw%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-01-10 16:55:09,893 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2573'}
2024-01-10 16:55:09,893 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2560] upload_file response body: 
2024-01-10 16:55:09,893 INFO    wandb-upload_0:2597697 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connectionpool.py", line 715, in urlopen
    httplib_response = self._make_request(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connectionpool.py", line 404, in _make_request
    self._validate_conn(conn)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1058, in _validate_conn
    conn.connect()
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connection.py", line 419, in connect
    self.sock = ssl_wrap_socket(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 449, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 493, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
ConnectionResetError: [Errno 104] Connection reset by peer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/adapters.py", line 486, in send
    resp = conn.urlopen(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connectionpool.py", line 799, in urlopen
    retries = retries.increment(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/util/retry.py", line 550, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/packages/six.py", line 769, in reraise
    raise value.with_traceback(tb)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connectionpool.py", line 715, in urlopen
    httplib_response = self._make_request(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connectionpool.py", line 404, in _make_request
    self._validate_conn(conn)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1058, in _validate_conn
    conn.connect()
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connection.py", line 419, in connect
    self.sock = ssl_wrap_socket(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 449, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 493, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2549, in upload_file
    response = self._upload_file_session.put(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/adapters.py", line 501, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2580, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2549, in upload_file
    response = self._upload_file_session.put(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/adapters.py", line 501, in send
    raise ConnectionError(err, request=request)
wandb.sdk.lib.retry.TransientError: None
2024-01-10 16:55:11,052 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:13,053 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:14,318 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/520/huggingface/4ffetev3/wandb-metadata.json?Expires=1704963306&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=I6KM711nl5VWVYhSgQZFEaH%2FWHWikDnRWjFVqNHXw%2BpUY5MNXLI4pMGpQEaYHSaVRX%2FDglipac46Z6uvIj6%2B6zO%2FKI6nbibE3uP1GcxYhpYQtMaop%2BQ5j4jD85%2B%2Brd97KCS9brRTziub7K602SkszBDBCphMYktmBkby3FJeceL0RvwEHQ8YBca%2FrPaNnX1rjTLIt68f2JHnRUw3CRIWFEg2QXrlS1h0%2BiPev3cJEVkppZ349bvZA5U1Jw%2FplFTVnTz3JELy%2BrNgbNmyqMtcfshccW1KzSoqu%2BVAmzKYjKni8uRhgAoPY9GR4jkqd6iFh%2FauSv7jzx%2FdhJlqCXTWgw%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-01-10 16:55:14,318 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2573'}
2024-01-10 16:55:14,318 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2560] upload_file response body: 
2024-01-10 16:55:14,618 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: status_report
2024-01-10 16:55:15,053 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:17,053 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:19,054 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:19,913 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: status_report
2024-01-10 16:55:21,055 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:21,232 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: stop_status
2024-01-10 16:55:21,232 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: stop_status
2024-01-10 16:55:21,235 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-10 16:55:23,055 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:24,255 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/520/huggingface/4ffetev3/wandb-metadata.json?Expires=1704963306&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=I6KM711nl5VWVYhSgQZFEaH%2FWHWikDnRWjFVqNHXw%2BpUY5MNXLI4pMGpQEaYHSaVRX%2FDglipac46Z6uvIj6%2B6zO%2FKI6nbibE3uP1GcxYhpYQtMaop%2BQ5j4jD85%2B%2Brd97KCS9brRTziub7K602SkszBDBCphMYktmBkby3FJeceL0RvwEHQ8YBca%2FrPaNnX1rjTLIt68f2JHnRUw3CRIWFEg2QXrlS1h0%2BiPev3cJEVkppZ349bvZA5U1Jw%2FplFTVnTz3JELy%2BrNgbNmyqMtcfshccW1KzSoqu%2BVAmzKYjKni8uRhgAoPY9GR4jkqd6iFh%2FauSv7jzx%2FdhJlqCXTWgw%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-01-10 16:55:24,255 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2573'}
2024-01-10 16:55:24,255 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2560] upload_file response body: 
2024-01-10 16:55:25,044 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: status_report
2024-01-10 16:55:25,056 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:27,056 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:29,057 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:30,667 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: status_report
2024-01-10 16:55:31,057 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:33,058 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:35,059 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:36,232 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: stop_status
2024-01-10 16:55:36,232 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: stop_status
2024-01-10 16:55:36,232 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-10 16:55:36,549 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: status_report
2024-01-10 16:55:37,059 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/config.yaml
2024-01-10 16:55:37,059 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:39,060 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:41,060 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:42,707 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: status_report
2024-01-10 16:55:43,061 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:43,672 DEBUG   SenderThread:2597697 [sender.py:send():380] send: exit
2024-01-10 16:55:43,672 INFO    SenderThread:2597697 [sender.py:send_exit():585] handling exit code: 1
2024-01-10 16:55:43,672 INFO    SenderThread:2597697 [sender.py:send_exit():587] handling runtime: 38
2024-01-10 16:55:43,672 INFO    SenderThread:2597697 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-01-10 16:55:43,672 INFO    SenderThread:2597697 [sender.py:send_exit():593] send defer
2024-01-10 16:55:43,672 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: defer
2024-01-10 16:55:43,672 INFO    HandlerThread:2597697 [handler.py:handle_request_defer():172] handle defer: 0
2024-01-10 16:55:43,672 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: defer
2024-01-10 16:55:43,673 INFO    SenderThread:2597697 [sender.py:send_request_defer():609] handle sender defer: 0
2024-01-10 16:55:43,673 INFO    SenderThread:2597697 [sender.py:transition_state():613] send defer: 1
2024-01-10 16:55:43,673 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: defer
2024-01-10 16:55:43,673 INFO    HandlerThread:2597697 [handler.py:handle_request_defer():172] handle defer: 1
2024-01-10 16:55:43,673 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: defer
2024-01-10 16:55:43,673 INFO    SenderThread:2597697 [sender.py:send_request_defer():609] handle sender defer: 1
2024-01-10 16:55:43,673 INFO    SenderThread:2597697 [sender.py:transition_state():613] send defer: 2
2024-01-10 16:55:43,673 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: defer
2024-01-10 16:55:43,673 INFO    HandlerThread:2597697 [handler.py:handle_request_defer():172] handle defer: 2
2024-01-10 16:55:43,673 INFO    HandlerThread:2597697 [system_monitor.py:finish():203] Stopping system monitor
2024-01-10 16:55:43,675 DEBUG   SystemMonitor:2597697 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-01-10 16:55:43,677 DEBUG   SystemMonitor:2597697 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-01-10 16:55:43,677 DEBUG   SystemMonitor:2597697 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-01-10 16:55:43,677 INFO    HandlerThread:2597697 [interfaces.py:finish():202] Joined cpu monitor
2024-01-10 16:55:43,678 INFO    HandlerThread:2597697 [interfaces.py:finish():202] Joined disk monitor
2024-01-10 16:55:43,890 INFO    HandlerThread:2597697 [interfaces.py:finish():202] Joined gpu monitor
2024-01-10 16:55:43,890 INFO    HandlerThread:2597697 [interfaces.py:finish():202] Joined memory monitor
2024-01-10 16:55:43,890 INFO    HandlerThread:2597697 [interfaces.py:finish():202] Joined network monitor
2024-01-10 16:55:43,891 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: defer
2024-01-10 16:55:43,891 INFO    SenderThread:2597697 [sender.py:send_request_defer():609] handle sender defer: 2
2024-01-10 16:55:43,891 INFO    SenderThread:2597697 [sender.py:transition_state():613] send defer: 3
2024-01-10 16:55:43,891 DEBUG   SenderThread:2597697 [sender.py:send():380] send: stats
2024-01-10 16:55:43,891 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: defer
2024-01-10 16:55:43,891 INFO    HandlerThread:2597697 [handler.py:handle_request_defer():172] handle defer: 3
2024-01-10 16:55:43,892 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: defer
2024-01-10 16:55:43,892 INFO    SenderThread:2597697 [sender.py:send_request_defer():609] handle sender defer: 3
2024-01-10 16:55:43,892 INFO    SenderThread:2597697 [sender.py:transition_state():613] send defer: 4
2024-01-10 16:55:43,892 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: defer
2024-01-10 16:55:43,892 INFO    HandlerThread:2597697 [handler.py:handle_request_defer():172] handle defer: 4
2024-01-10 16:55:43,892 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: defer
2024-01-10 16:55:43,892 INFO    SenderThread:2597697 [sender.py:send_request_defer():609] handle sender defer: 4
2024-01-10 16:55:43,892 INFO    SenderThread:2597697 [sender.py:transition_state():613] send defer: 5
2024-01-10 16:55:43,892 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: defer
2024-01-10 16:55:43,892 INFO    HandlerThread:2597697 [handler.py:handle_request_defer():172] handle defer: 5
2024-01-10 16:55:43,892 DEBUG   SenderThread:2597697 [sender.py:send():380] send: summary
2024-01-10 16:55:43,892 INFO    SenderThread:2597697 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-01-10 16:55:43,893 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: defer
2024-01-10 16:55:43,893 INFO    SenderThread:2597697 [sender.py:send_request_defer():609] handle sender defer: 5
2024-01-10 16:55:43,893 INFO    SenderThread:2597697 [sender.py:transition_state():613] send defer: 6
2024-01-10 16:55:43,893 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: defer
2024-01-10 16:55:43,893 INFO    HandlerThread:2597697 [handler.py:handle_request_defer():172] handle defer: 6
2024-01-10 16:55:43,893 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: defer
2024-01-10 16:55:43,893 INFO    SenderThread:2597697 [sender.py:send_request_defer():609] handle sender defer: 6
2024-01-10 16:55:43,893 INFO    SenderThread:2597697 [sender.py:transition_state():613] send defer: 7
2024-01-10 16:55:43,893 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: status_report
2024-01-10 16:55:43,893 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: defer
2024-01-10 16:55:43,893 INFO    HandlerThread:2597697 [handler.py:handle_request_defer():172] handle defer: 7
2024-01-10 16:55:43,893 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: defer
2024-01-10 16:55:43,893 INFO    SenderThread:2597697 [sender.py:send_request_defer():609] handle sender defer: 7
2024-01-10 16:55:44,051 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/520/huggingface/4ffetev3/wandb-metadata.json?Expires=1704963306&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=I6KM711nl5VWVYhSgQZFEaH%2FWHWikDnRWjFVqNHXw%2BpUY5MNXLI4pMGpQEaYHSaVRX%2FDglipac46Z6uvIj6%2B6zO%2FKI6nbibE3uP1GcxYhpYQtMaop%2BQ5j4jD85%2B%2Brd97KCS9brRTziub7K602SkszBDBCphMYktmBkby3FJeceL0RvwEHQ8YBca%2FrPaNnX1rjTLIt68f2JHnRUw3CRIWFEg2QXrlS1h0%2BiPev3cJEVkppZ349bvZA5U1Jw%2FplFTVnTz3JELy%2BrNgbNmyqMtcfshccW1KzSoqu%2BVAmzKYjKni8uRhgAoPY9GR4jkqd6iFh%2FauSv7jzx%2FdhJlqCXTWgw%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-01-10 16:55:44,052 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2573'}
2024-01-10 16:55:44,052 ERROR   wandb-upload_0:2597697 [internal_api.py:upload_file():2560] upload_file response body: 
2024-01-10 16:55:44,061 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_created():271] file/dir created: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/wandb-summary.json
2024-01-10 16:55:44,672 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:55:44,986 INFO    SenderThread:2597697 [sender.py:transition_state():613] send defer: 8
2024-01-10 16:55:44,986 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:55:44,987 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: defer
2024-01-10 16:55:44,987 INFO    HandlerThread:2597697 [handler.py:handle_request_defer():172] handle defer: 8
2024-01-10 16:55:44,987 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: defer
2024-01-10 16:55:44,987 INFO    SenderThread:2597697 [sender.py:send_request_defer():609] handle sender defer: 8
2024-01-10 16:55:44,987 INFO    SenderThread:2597697 [job_builder.py:build():281] Attempting to build job artifact
2024-01-10 16:55:44,989 INFO    SenderThread:2597697 [job_builder.py:_get_source_type():404] no source found
2024-01-10 16:55:44,989 INFO    SenderThread:2597697 [sender.py:transition_state():613] send defer: 9
2024-01-10 16:55:44,989 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: defer
2024-01-10 16:55:44,989 INFO    HandlerThread:2597697 [handler.py:handle_request_defer():172] handle defer: 9
2024-01-10 16:55:44,990 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: defer
2024-01-10 16:55:44,990 INFO    SenderThread:2597697 [sender.py:send_request_defer():609] handle sender defer: 9
2024-01-10 16:55:44,990 INFO    SenderThread:2597697 [dir_watcher.py:finish():358] shutting down directory watcher
2024-01-10 16:55:45,061 INFO    Thread-12 :2597697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:45,062 INFO    SenderThread:2597697 [dir_watcher.py:finish():388] scan: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files
2024-01-10 16:55:45,062 INFO    SenderThread:2597697 [dir_watcher.py:finish():402] scan save: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/wandb-summary.json wandb-summary.json
2024-01-10 16:55:45,062 INFO    SenderThread:2597697 [dir_watcher.py:finish():402] scan save: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/wandb-metadata.json wandb-metadata.json
2024-01-10 16:55:45,062 INFO    SenderThread:2597697 [dir_watcher.py:finish():402] scan save: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/requirements.txt requirements.txt
2024-01-10 16:55:45,064 INFO    SenderThread:2597697 [dir_watcher.py:finish():402] scan save: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/conda-environment.yaml conda-environment.yaml
2024-01-10 16:55:45,066 INFO    SenderThread:2597697 [dir_watcher.py:finish():402] scan save: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log output.log
2024-01-10 16:55:45,066 INFO    SenderThread:2597697 [dir_watcher.py:finish():402] scan save: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/config.yaml config.yaml
2024-01-10 16:55:45,069 INFO    SenderThread:2597697 [sender.py:transition_state():613] send defer: 10
2024-01-10 16:55:45,069 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: defer
2024-01-10 16:55:45,070 INFO    HandlerThread:2597697 [handler.py:handle_request_defer():172] handle defer: 10
2024-01-10 16:55:45,071 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: defer
2024-01-10 16:55:45,071 INFO    SenderThread:2597697 [sender.py:send_request_defer():609] handle sender defer: 10
2024-01-10 16:55:45,071 INFO    SenderThread:2597697 [file_pusher.py:finish():175] shutting down file pusher
2024-01-10 16:55:45,354 ERROR   wandb-upload_2:2597697 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/520/huggingface/4ffetev3/requirements.txt?Expires=1704963345&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=W4eX%2FBwii6IHwnWuempkUDj5Xly%2FVuiBxT1cnZL6kfyxLXQxMrKXVyFYJJoU%2BF015PcQxJnZN1EUEhjt444Kx7BRB4F7Y2HxwHd9mLVt9TPac8G8fn4MuiUqxURVDy86Z0QjroZdNwXLYSa3A4xZdhcwNXv15eHBJ7IkyEoJOCMw48YNvKieFMXuxPmfqKiK0OdPRWGT7%2FkkGOrzAdb9KHeZhLntmiCjLQIGTkGytxDmfLqfHeSWucTTHdEN%2F15AoBZ7cUSYkUdM98aoc%2FhEVLR3fxai6VXVDXjSMaeYZ7%2F2Ij1aSxCWLx42aqDVp32oXygm1iiDdiZoWOuImc2Zhw%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-01-10 16:55:45,355 ERROR   wandb-upload_2:2597697 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5737'}
2024-01-10 16:55:45,355 ERROR   wandb-upload_2:2597697 [internal_api.py:upload_file():2560] upload_file response body: 
2024-01-10 16:55:45,390 ERROR   wandb-upload_1:2597697 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/520/huggingface/4ffetev3/wandb-summary.json?Expires=1704963345&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=gFK2XM2DaKO7WUA5rYy335%2Bh%2BwSek72mBT5N6TU1Rk5fBOdZ9HKfjzPSMX4dfpWs5rtMHDCBPeiFNA3nhi9lg7bXZrEm1KIhh1ge89ZWFFwBbspIDroW0OIOvQqg%2FBu9vYBEdY9tolXcpVQruONhGLlP3BBE18yNwtHvv0Q1x7RektzxVZY%2FeJABdWyQ7IT8CJ5du1DHXPQ%2BXRepIgq1HIwRZArFUGfzSdXbg3uy2%2BybRjHOTbn4A%2F0LzuQwP9S%2FX2I1afsCELUxBVyRfc6L95BFt0QMguSAxNCfu1n1zbLpC942rYU25HF6LPP9CkQIiPLUP3H4YCWSmcmHfjXgHg%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-01-10 16:55:45,390 ERROR   wandb-upload_1:2597697 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '27'}
2024-01-10 16:55:45,390 ERROR   wandb-upload_1:2597697 [internal_api.py:upload_file():2560] upload_file response body: 
2024-01-10 16:55:45,672 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:55:45,673 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:55:45,702 ERROR   wandb-upload_3:2597697 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/520/huggingface/4ffetev3/conda-environment.yaml?Expires=1704963345&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=MbDypsvfMw0lfP4HApzl5sU5I0Of%2FId8H9pTIWm0FKFYQuE5zRLwjcBHjc8BnCsiihaigpypBTI0KTAtbfztugaDtLSw0%2Fk3WEzsh9Jr2YNAbFPPCbvrE5qsYhMF%2FmDYwIvDvbbBiWf2R5NVJBKDhQhxNBaSpBeu60nQDmBUeJDhrsFxBQGZU5of9oTqSfEFR%2FuRR4n9Vz8j2dXWkF9yt%2Fhlj6S1XdopUgdSXbQdcwvh0Te3pQ4UBV9px4ruNbskK2fFmhZnXTmXSwZrHWqQ5Dc%2FWLSPvlTUniSI4hRYkq7gPsonrkcvULZy8nLlvZOrKXdjcYEtaoD8btoPPi5fMQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-01-10 16:55:45,702 ERROR   wandb-upload_3:2597697 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '260'}
2024-01-10 16:55:45,702 ERROR   wandb-upload_3:2597697 [internal_api.py:upload_file():2560] upload_file response body: 
2024-01-10 16:55:45,702 ERROR   wandb-upload_4:2597697 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/520/huggingface/4ffetev3/output.log?Expires=1704963345&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=eh6CTmDyTRVjNQuKwKWTAGmgllXmfzyCG00VRVaeb9xmCRyR6ujsg5ugAFzImNIZB4W53j34HE2Y94G9RbW6LtLzY0EstdwhBciqZ227vT6laBAHVSQCJdsNJaP6uM36NaKNwi9k2EjMiJhf7EjFgjw67NdWORlQbdXS54JpnnDuf%2F5ZRNmgAxo7OcLeUaeaYerYHAVWefY5CvN9TCnREt2CHk7%2BUvPELHucXYzKDTL1uqVDRh5aVfZmTb6D88NfcB8x4eIuf9I9TU%2BwrJ1IlSUvqHmbcc4n9icXO8KLbM6hdjOQ45yoFb2NK2RV33vw4Au9lfh5OyTI5v86wUx4zg%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-01-10 16:55:45,702 ERROR   wandb-upload_4:2597697 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5729'}
2024-01-10 16:55:45,702 ERROR   wandb-upload_4:2597697 [internal_api.py:upload_file():2560] upload_file response body: 
2024-01-10 16:55:46,129 INFO    wandb-upload_5:2597697 [upload_job.py:push():131] Uploaded file /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/config.yaml
2024-01-10 16:55:46,524 ERROR   wandb-upload_1:2597697 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/520/huggingface/4ffetev3/wandb-summary.json?Expires=1704963345&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=gFK2XM2DaKO7WUA5rYy335%2Bh%2BwSek72mBT5N6TU1Rk5fBOdZ9HKfjzPSMX4dfpWs5rtMHDCBPeiFNA3nhi9lg7bXZrEm1KIhh1ge89ZWFFwBbspIDroW0OIOvQqg%2FBu9vYBEdY9tolXcpVQruONhGLlP3BBE18yNwtHvv0Q1x7RektzxVZY%2FeJABdWyQ7IT8CJ5du1DHXPQ%2BXRepIgq1HIwRZArFUGfzSdXbg3uy2%2BybRjHOTbn4A%2F0LzuQwP9S%2FX2I1afsCELUxBVyRfc6L95BFt0QMguSAxNCfu1n1zbLpC942rYU25HF6LPP9CkQIiPLUP3H4YCWSmcmHfjXgHg%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-01-10 16:55:46,524 ERROR   wandb-upload_1:2597697 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '27'}
2024-01-10 16:55:46,524 ERROR   wandb-upload_1:2597697 [internal_api.py:upload_file():2560] upload_file response body: 
2024-01-10 16:55:46,524 INFO    wandb-upload_1:2597697 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connectionpool.py", line 715, in urlopen
    httplib_response = self._make_request(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connectionpool.py", line 404, in _make_request
    self._validate_conn(conn)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1058, in _validate_conn
    conn.connect()
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connection.py", line 419, in connect
    self.sock = ssl_wrap_socket(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 449, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 493, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
ConnectionResetError: [Errno 104] Connection reset by peer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/adapters.py", line 486, in send
    resp = conn.urlopen(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connectionpool.py", line 799, in urlopen
    retries = retries.increment(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/util/retry.py", line 550, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/packages/six.py", line 769, in reraise
    raise value.with_traceback(tb)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connectionpool.py", line 715, in urlopen
    httplib_response = self._make_request(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connectionpool.py", line 404, in _make_request
    self._validate_conn(conn)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1058, in _validate_conn
    conn.connect()
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/connection.py", line 419, in connect
    self.sock = ssl_wrap_socket(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 449, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 493, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2549, in upload_file
    response = self._upload_file_session.put(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/adapters.py", line 501, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2580, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2549, in upload_file
    response = self._upload_file_session.put(
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/shangjian/miniconda3/envs/pro/lib/python3.8/site-packages/requests/adapters.py", line 501, in send
    raise ConnectionError(err, request=request)
wandb.sdk.lib.retry.TransientError: None
2024-01-10 16:55:46,673 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:55:46,674 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:55:46,720 INFO    wandb-upload_2:2597697 [upload_job.py:push():131] Uploaded file /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/requirements.txt
2024-01-10 16:55:46,963 ERROR   wandb-upload_3:2597697 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/520/huggingface/4ffetev3/conda-environment.yaml?Expires=1704963345&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=MbDypsvfMw0lfP4HApzl5sU5I0Of%2FId8H9pTIWm0FKFYQuE5zRLwjcBHjc8BnCsiihaigpypBTI0KTAtbfztugaDtLSw0%2Fk3WEzsh9Jr2YNAbFPPCbvrE5qsYhMF%2FmDYwIvDvbbBiWf2R5NVJBKDhQhxNBaSpBeu60nQDmBUeJDhrsFxBQGZU5of9oTqSfEFR%2FuRR4n9Vz8j2dXWkF9yt%2Fhlj6S1XdopUgdSXbQdcwvh0Te3pQ4UBV9px4ruNbskK2fFmhZnXTmXSwZrHWqQ5Dc%2FWLSPvlTUniSI4hRYkq7gPsonrkcvULZy8nLlvZOrKXdjcYEtaoD8btoPPi5fMQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-01-10 16:55:46,964 ERROR   wandb-upload_3:2597697 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '260'}
2024-01-10 16:55:46,964 ERROR   wandb-upload_3:2597697 [internal_api.py:upload_file():2560] upload_file response body: 
2024-01-10 16:55:47,141 INFO    wandb-upload_4:2597697 [upload_job.py:push():131] Uploaded file /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/output.log
2024-01-10 16:55:47,674 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:55:47,674 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:55:48,675 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:55:48,676 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:55:48,867 INFO    wandb-upload_1:2597697 [upload_job.py:push():131] Uploaded file /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/wandb-summary.json
2024-01-10 16:55:49,325 INFO    wandb-upload_3:2597697 [upload_job.py:push():131] Uploaded file /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/files/conda-environment.yaml
2024-01-10 16:55:49,676 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:55:49,676 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:55:50,677 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:55:50,677 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:55:51,677 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:55:51,678 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:55:52,678 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:55:52,678 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:55:53,679 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:55:53,679 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: status_report
2024-01-10 16:55:53,679 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:55:54,680 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:55:54,680 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:55:55,681 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:55:55,681 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:55:56,682 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:55:56,682 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:55:57,683 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:55:57,683 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:55:58,684 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: status_report
2024-01-10 16:55:58,684 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:55:58,684 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:55:59,685 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:55:59,685 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:00,685 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:00,686 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:01,686 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:01,687 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:02,687 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:02,687 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:03,688 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:03,688 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: status_report
2024-01-10 16:56:03,688 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:04,688 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:04,689 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:05,689 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:05,689 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:06,690 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:06,690 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:07,691 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:07,691 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:08,691 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: status_report
2024-01-10 16:56:08,692 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:08,692 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:09,692 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:09,693 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:10,693 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:10,694 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:11,694 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:11,694 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:12,694 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:12,695 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:13,695 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: status_report
2024-01-10 16:56:13,695 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:13,696 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:14,696 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:14,697 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:15,697 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:15,697 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:16,698 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:16,698 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:17,698 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:17,699 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:18,699 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: status_report
2024-01-10 16:56:18,699 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:18,700 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:19,700 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:19,700 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:20,352 INFO    wandb-upload_0:2597697 [upload_job.py:push():131] Uploaded file /tmp/tmps2z4v3zfwandb/dcs8u3er-wandb-metadata.json
2024-01-10 16:56:20,552 INFO    Thread-11 :2597697 [sender.py:transition_state():613] send defer: 11
2024-01-10 16:56:20,552 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: defer
2024-01-10 16:56:20,552 INFO    HandlerThread:2597697 [handler.py:handle_request_defer():172] handle defer: 11
2024-01-10 16:56:20,553 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: defer
2024-01-10 16:56:20,553 INFO    SenderThread:2597697 [sender.py:send_request_defer():609] handle sender defer: 11
2024-01-10 16:56:20,553 INFO    SenderThread:2597697 [file_pusher.py:join():181] waiting for file pusher
2024-01-10 16:56:20,553 INFO    SenderThread:2597697 [sender.py:transition_state():613] send defer: 12
2024-01-10 16:56:20,553 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: defer
2024-01-10 16:56:20,553 INFO    HandlerThread:2597697 [handler.py:handle_request_defer():172] handle defer: 12
2024-01-10 16:56:20,553 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: defer
2024-01-10 16:56:20,553 INFO    SenderThread:2597697 [sender.py:send_request_defer():609] handle sender defer: 12
2024-01-10 16:56:20,553 INFO    SenderThread:2597697 [file_stream.py:finish():595] file stream finish called
2024-01-10 16:56:20,701 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:20,814 INFO    SenderThread:2597697 [file_stream.py:finish():599] file stream finish is done
2024-01-10 16:56:20,815 INFO    SenderThread:2597697 [sender.py:transition_state():613] send defer: 13
2024-01-10 16:56:20,815 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:20,815 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: defer
2024-01-10 16:56:20,815 INFO    HandlerThread:2597697 [handler.py:handle_request_defer():172] handle defer: 13
2024-01-10 16:56:20,815 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: defer
2024-01-10 16:56:20,815 INFO    SenderThread:2597697 [sender.py:send_request_defer():609] handle sender defer: 13
2024-01-10 16:56:20,815 INFO    SenderThread:2597697 [sender.py:transition_state():613] send defer: 14
2024-01-10 16:56:20,816 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: defer
2024-01-10 16:56:20,816 DEBUG   SenderThread:2597697 [sender.py:send():380] send: final
2024-01-10 16:56:20,816 INFO    HandlerThread:2597697 [handler.py:handle_request_defer():172] handle defer: 14
2024-01-10 16:56:20,816 DEBUG   SenderThread:2597697 [sender.py:send():380] send: footer
2024-01-10 16:56:20,816 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: defer
2024-01-10 16:56:20,816 INFO    SenderThread:2597697 [sender.py:send_request_defer():609] handle sender defer: 14
2024-01-10 16:56:20,816 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:20,816 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:20,817 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-10 16:56:20,817 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: poll_exit
2024-01-10 16:56:20,817 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: server_info
2024-01-10 16:56:20,817 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: server_info
2024-01-10 16:56:20,817 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: get_summary
2024-01-10 16:56:20,819 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: sampled_history
2024-01-10 16:56:20,819 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-10 16:56:20,819 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: job_info
2024-01-10 16:56:21,037 DEBUG   SenderThread:2597697 [sender.py:send_request():407] send_request: job_info
2024-01-10 16:56:21,037 INFO    MainThread:2597697 [wandb_run.py:_footer_history_summary_info():3599] rendering history
2024-01-10 16:56:21,038 INFO    MainThread:2597697 [wandb_run.py:_footer_history_summary_info():3631] rendering summary
2024-01-10 16:56:21,038 INFO    MainThread:2597697 [wandb_run.py:_footer_sync_info():3558] logging synced files
2024-01-10 16:56:21,038 DEBUG   HandlerThread:2597697 [handler.py:handle_request():146] handle_request: shutdown
2024-01-10 16:56:21,038 INFO    HandlerThread:2597697 [handler.py:finish():866] shutting down handler
2024-01-10 16:56:21,820 INFO    WriterThread:2597697 [datastore.py:close():294] close: /home/shangjian/code/Research/Multimodal & LLM/SLM/wandb/run-20240110_165503-4ffetev3/run-4ffetev3.wandb
2024-01-10 16:56:22,037 INFO    SenderThread:2597697 [sender.py:finish():1534] shutting down sender
2024-01-10 16:56:22,038 INFO    SenderThread:2597697 [file_pusher.py:finish():175] shutting down file pusher
2024-01-10 16:56:22,038 INFO    SenderThread:2597697 [file_pusher.py:join():181] waiting for file pusher
